mayastor/src/nexus_uri.rs

```diff
@@ -53,6 +53,12 @@ pub enum NexusBdevError {
     // Bdev create/destroy errors
     #[snafu(display("bdev {} already exists", name))]
     BdevExists { name: String },
+    #[snafu(display(
+        "bdev {} already exists with a different uuid: {}",
```

Comment:
Does the bdev name contain the uuid within it? If not, it could be useful to output both the requested uuid and the existing uuid.